pkgs/development/libraries/ncurses/default.nix

```diff
@@ -35,6 +36,16 @@ stdenv.mkDerivation rec {
     "--without-debug"
     "--enable-pc-files"
     "--enable-symlinks"
+  ] ++ lib.optionals androidMinimal [
+    "--without-cxx"
+    #"--host=arm"
+    "--enable-static"
+    #"--disable-shared"
+    #"--without-manpages"
+    #"--without-debug"
+    #"--without-termlib"
+    #"--without-ticlib"
+    "--without-cxx"
```

Comment:
Probably want to remove all these commented-out lines, and de-duplicate what remains.

Just curious, why "--without-cxx"?

Reply:
literally cargo culting some prior GHC on android work a year ago. I was too busy hacking up the first draft of everything we've merged since to look into this.